README.adoc

```diff
@@ -8,42 +8,430 @@ The time for the full course is around three to four days.
 
 The material is created with people with zero Rust experience but with a programming background in mind.
 
-## Overview
+Ferrous Systems offers a large Rust curriculum for both beginner and
+advanced Rust developer at all levels. We are the oldest and most often
+chosen Rust training provider.
+
+Ferrous Systems specialises in custom, topic focused workshops for
+enterprises starting moving into Rust. Supplemental to our courses, we
+offer ongoing help and feedback.
+
+Overview
+--------
 
 The materials are presented as a set of small, self-contained lessons on a specific topic.
+Note that lessons might be taken down when deemed outdated and recreated for new audiences.
 Lessons can be arranged into courses exploring various aspects of Rust.
 
-## Building
+Pricing
+~~~~~~~
+
+Ferrous Systems usually charges per trainer day. That allows for easy
+calculation on one-off trainings and a fair and easy discussion about
```

Comment:
This paragraph seems to exist to justify our pricing scheme, which I don't think is necessary.

I would replace with something like:

> For standard trainings, Ferrous Systems charges on a per-trainer-per-day basis. Quotations for custom trainings are available on request.

Reply:
Applied manually.